handlers/main.yml

```diff
@@ -1,5 +1,20 @@
 # handlers file for postfix
 ---
+- name: new aliases
+  command: newaliases
+
+- name: new virtual aliases
+  command: postmap /etc/postfix/virtual
+
+- name: postmap sasl_passwd
+  command: postmap hash:/etc/postfix/sasl_passwd
```

Comment:
@tersmitten `hash:` not needed?

Reply:
http://www.postfix.org/postmap.1.html

Reply:
I see, it is optional. Probably good to make it consistent with line 7?